app/assets/javascripts/govuk_publishing_components/lib/cookie-settings.js

```diff
@@ -107,7 +107,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     try {
       documentReferrer = document.referrer || new URL(document.referrer).pathname
     } catch (e) {
-      console.error('Error grabbing referrer for cookie settings', window.location, e)
+      console.warn('Error grabbing referrer for cookie settings', window.location, e)
```

Comment:
I've changed this to warn, otherwise it makes the cookie page look like the JS is crashing when you visit the page without a referrer.